task/sanity-label-check/0.1/sanity-label-check.yaml

```diff
@@ -35,7 +37,8 @@ spec:
         . /utils.sh
         if [ ! -s ../sanity-inspect-image/image_inspect.json ]; then
           echo "File ../sanity-inspect-image/image_inspect.json is not generated correctly, please check HACBS_TEST_OUTPUT of task sanity-inspect-image"
-          HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -t 'File ../sanity-inspect-image/image_inspect.json is not generated correctly, please check HACBS_TEST_OUTPUT of task sanity-inspect-image!')"
+          note="Task $(context.task.name) meets error: File ../sanity-inspect-image/image_inspect.json is not generated correctly, please check HACBS_TEST_OUTPUT of task sanity-inspect-image!"
```

Comment:
This relative path with `..` inside is not very user friendly, can we use full path?

Reply:
Updated to use the full path.